canonical/anbox-cloud.io
# Remove all flask search references in favour of RTD

## Done

- Upgraded webteam.flask-base version
- Removed webteam.search and webteam.discourse module since we moved to read the docs in #350 
- Removed all unused markup, styles and python functions for search.

## QA

- Check out the demo site: https://anbox-cloud-io-355.demos.haus/
- Check if search works well and you're redirected to documentation.u.c.
- Check if anything is unexpected related to search.



## Issue / Card

Fixes [WD-15028](https://warthogs.atlassian.net/browse/WD-15028?atlOrigin=eyJpIjoiOTRkOTUwNzA3Mjc3NGZhYzgwNTFmMGUzNzA2M2ZiOTUiLCJwIjoiaiJ9)




[WD-15028]: https://warthogs.atlassian.net/browse/WD-15028?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
